models/datatypes/NcOrganizationId.json

```diff
@@ -0,0 +1,7 @@
+{
+    "description": "Unique 24-bit organization ID",
```

Comment:
Consistency with capitalization of ID/id. Lower case for "Object id" so should be lower case here for consistency?